### PR TITLE
Merge PRs

### PR DESCRIPTION
**Pull request by saraholiviawu**
Merge PRs. Resolve merge conflicts.

### CODE REVIEW

File: ripple/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+    content = `
+      <ul class="navbar-nav">
+        <li class="nav-item active">
+          <a class="nav-link login-chip" href="login.html">Log in</a>
+        </li>
+      </ul>
+    `;
+    $("#today").addClass("today-padding");
+  } else if (userType == "false") { // Community member: avatar + 2 item drop down
+  console.log("entered userType false loop");
```
Comment (peter14f) [inline ripple/src/main/webapp/script.js:23]: fix indentation

File: ripple/src/main/webapp/script.js [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+              aria-labelledby="navbarDropdownMenuLink-55">
+            <a class="dropdown-item" href="accountsettings.html">Account settings</a>
+            <div class="dropdown-divider"></div>
+            <a class="dropdown-item semi-bold-text" onclick="signOutUser()">Log out</a>
+          </div>
+        </li>
+      </ul>
+    `;
+  } else if (userType == "true") { // Business owner: avatar + 3 item drop down
+  console.log("entered userType true loop");
```
Comment (peter14f) [inline ripple/src/main/webapp/script.js:41]: same

Comment (saraholiviawu) [reply ripple/src/main/webapp/script.js:41]: Fixed.

File: ripple/src/main/webapp/script.js (hunk 1 above)
Comment (saraholiviawu) [reply ripple/src/main/webapp/script.js:23]: Fixed.

**Pull request merged**